ingestion/src/metadata/ingestion/source/dashboard/metabase.py

```diff
@@ -215,7 +215,7 @@ def yield_dashboard_lineage_details(
                         )
                         from_entities = search_table_entities(
                             metadata=self.metadata,
-                            database=database["details"]["dbname"],
+                            database=database["details"]["db"],
```

Comment:
does this change make this compatible with different versions? What version were we currently supporting?

Reply:
Hey @pmbrull we have metabae v0.42.4 and I found that we are not getting linage because of this key, so I did this change and now we are getting linage.
User has metabase version -- 0.43.4 so I will test with that version .

Reply:
aaah I see thanks, then it was a more generic issue. LGTM!

Could we also pls document the metabase version on the docs? Might be useful for some users to know where we tested this

Reply:
Sure I will update the doc for that.